.travis.yml

```diff
@@ -39,4 +39,4 @@ before_install:
 # library, executables, and test suites, and runs the test
 # suites. --no-terminal works around some quirks in Travis's terminal
 # implementation.
-script: stack $ARGS --no-terminal --install-ghc test
+script: stack $ARGS --no-terminal --install-ghc test wai-middleware-throttle
```

Comment:
I have new circle ci config I've been using: https://gist.github.com/creichert/efb84716d628c431e827ff95289b6aa7

You can pull it in here, if you want (else I'll do it). It'll test the change against several lts and nightly builds and it's fast.

Reply:
Just pushed it, I'll get all the builds working and merge it in here.